include/lib/zlib/gunzip.h

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
I do not have a strong opinion.

If you want to distinguish TF-specific files, I can rename the C file to tf_gunzip.c.

The directory include/lib/zlib/ does not contain any file from upstream.
If you still it is better to distinguish it (from what?), I can rename the header to tf_gunzip.h as well.

Reply:
I think Jeenu's comment is more to do with gunzip.c, which does live with the other upstream files in lib/zlib. It's not so important to rename gunzip.h, but it might be nice if the source and header had the same base name.